modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java

```diff
@@ -49,6 +49,47 @@
 public class OpenAPIDeserializerTest {
 
     @Test
+    public void securityTest(@Injectable List<AuthorizationValue> auths){
+        String yaml = "openapi: \"3.0.0\"\n" +
+                "info:\n" +
+                "  title: Test OpenApi Specification\n" +
+                "  version: 1.0.0\n" +
+                "paths:\n" +
+                "  /test:\n" +
+                "    get:\n" +
+                "      operationId: testGet\n" +
+                "      security:\n" +
+                "      - openId:\n" +
+                "        - https://scopes.example.com/myScope\n" +
```

Comment:
As scope is used for access control of API. Above scope is not good example of "scope". 
You can take sample value from original Open API 3 spec. 
For example:
```yaml
openId_auth:
- write:pets
- read:pets
```
or
```
openId_auth:
- write
- read
```

Reply:
I also added a test following this advice. Thanks!

Reply:
Great, just one last question. Why not updated "https://scopes.example.com/myScope" to reflect real world scope example. :)

Reply:
that's actually a real world example jeje. from this ticket https://github.com/swagger-api/swagger-parser/issues/562 :)